valiu-node-rpc/src/valiu_extra.rs

```diff
@@ -0,0 +1,179 @@
+use core::{fmt, marker::PhantomData};
```

Comment:
I'm fine with this custom "`Extra`" implementation but would you say the weird unknown errors are bugs in any of the dependencies or something we should add a note and possibly create issues somewhere to track for future reference and hopefully remove this custom implementation some day?

Reply:
Indeed, this feature should be noted somewhere since Substrate rejects extrinsics sent using the default extra parameters.
I personally don't know how to remove this custom `Extra` without defaulting back to the original type. Maybe there is a Substrate parameter that deals with it but I am not sure.

Reply:
Then if is not to much to ask could file an issue in substrate or the relevant repo and add a `// NOTE: <link>` here for future reference? in a few months it would be forgotten this is a "hack" and just remain there forever as part of the landscape 😜

Reply:
Hum... I am sorry for the misunderstanding, Substrate is not the one to blame here because it is only trying to de-serialize what was defined in the runtime. The truth is: `DefaultExtra` of `substrate-subxt` contains `TransactionPayment` but our runtime doesn't expect it because `pallet-balance` was  removed, thus the need to define our own `ValiuExtra` to successfully send extrinsics. It is more like a feature then a "hack".

To remove some burden on our side, I will try to re-export all these duplicated `Extra` types ìn `substrate-subxt`.

Reply:
Ok now I get it! 👍🏽 and sure feel free to make it look prettier :)